web-bundle/src/main/java/com/graphhopper/http/GraphHopperManaged.java

```diff
@@ -87,6 +93,27 @@ public GraphHopperManaged(GraphHopperConfig configuration, ObjectMapper objectMa
             SpatialRuleLookupHelper.buildAndInjectCountrySpatialRules(graphHopper,
                     new Envelope(maxBounds.minLon, maxBounds.maxLon, maxBounds.minLat, maxBounds.maxLat), jsonFeatureCollections);
         }
+
+        ObjectMapper yamlOM = Jackson.initObjectMapper(new ObjectMapper(new YAMLFactory()));
+        List<ProfileConfig> newProfiles = new ArrayList<>();
+        for (ProfileConfig profileConfig : configuration.getProfiles()) {
+            if (!CustomWeighting.NAME.equals(profileConfig.getWeighting())) {
+                newProfiles.add(profileConfig);
+                continue;
+            }
+            String customModelLocation = profileConfig.getHints().getString("custom_model_file", "");
+            if (customModelLocation.isEmpty())
+                newProfiles.add(new CustomProfileConfig(profileConfig).setCustomModel(new CustomModel()));
```

Comment:
We cannot specify the custom model properties directly in the profiles in `config.yml`, right? But we *can* skip `custom_model_file` to get an 'empty' model which we then can use to send custom models per request? Shall we make this a bit more explicit? Something like enforcing `custom_model_file=per_request` to do this or something? Otherwise if someone specifies some custom model stuff in the profile it will just be ignored / replaced by `new CustomModel()` here, which might be unexpected?

Reply:
Hmmh, indeed and with `custom_model_file=empty` it might be fine until we allow an embedded model.